lib/api/update/photo.ts

```diff
@@ -6,7 +6,8 @@ import smartcrop from 'smartcrop-sharp';
 import to from 'await-to-js';
 import { v4 as uuid } from 'uuid';
 
-import { Account, Constructor } from 'lib/model';
+import { Account } from 'lib/model/account';
+import { Constructor } from 'lib/model/resource';
```

Comment:
Sort imports (in this case, alphabetically: the `APIError` import should occur before the `Constructor` import).